rep-2005.rst

```diff
@@ -0,0 +1,235 @@
+    * `ros-perception/image_transport_plugins <https://github.com/ros-perception/image_transport_plugins>`_
+    * `ros-perception/perception_pcl <https://github.com/ros-perception/perception_pcl>`_
+    * `ros-perception/vision_opencv <https://github.com/ros-perception/vision_opencv>`_
+    * `ros-perception/laser_filters <https://github.com/ros-perception/laser_filters>`_
+    * `ros-perception/laser_geometry <https://github.com/ros-perception/laser_geometry>`_
+
+  * `ros-planning/navigation2 <https://github.com/ros-planning/navigation2>`_
+  * `ros-planning/moveit2 <https://github.com/ros-planning/moveit2>`_
+  * `ros-visualization/interactive_markers <https://github.com/ros-visualization/interactive_markers>`_
+  * `ros2/geometry2 <https://github.com/ros2/geometry2>`_
+  * `ros2/kdl_parser <https://github.com/ros2/kdl_parser>`_
+  * `ros2/message_filters <https://github.com/ros2/message_filters>`_
+  * `ros2/sros2 <https://github.com/ros2/sros2>`_
+  * `ros2/teleop_twist_joy <https://github.com/ros2/teleop_twist_joy>`_
+  * ROS Drivers
+
+    * `ros2/joystick_drivers <https://github.com/ros2/joystick_drivers>`_
+    * `ros-drivers/velodyne <https://github.com/ros-drivers/velodyne>`_
+    * `ros-drivers/urg_c <https://github.com/ros-drivers/urg_c>`_
+    * `ros-drivers/urg_node <https://github.com/ros-drivers/urg_node>`_
+
+* Documentation, Examples, Tutorials
+
+  * `ros2/demos <https://github.com/ros2/demos>`_
+  * `ros2/design <https://github.com/ros2/design>`_
+  * `ros2/examples <https://github.com/ros2/examples>`_
+  * `ros2/ros_core_documentation <https://github.com/ros2/ros_core_documentation>`_
+  * `ros2/ros2_documentation <https://github.com/ros2/ros2_documentation>`_
+
+* Robot
+
+  * `ros/urdfdom_headers <https://github.com/ros/urdfdom_headers>`_
+  * `ros2/urdf <https://github.com/ros2/urdf>`_
+  * `ros2/urdfdom <https://github.com/ros2/urdfdom>`_
+
+* Tools
+
+  * `ros-simulation/gazebo_ros_pkgs <https://github.com/ros-simulation/gazebo_ros_pkgs>`_
+  * `ros2/ros1_bridge <https://github.com/ros2/ros1_bridge>`_
+  * `ros2/ros2cli <https://github.com/ros2/ros2cli>`_
+  * `ros2/rosbag2 <https://github.com/ros2/rosbag2>`_
+  * `ros2/rviz <https://github.com/ros2/rviz>`_
+  * `ros-tooling/cross_compile <https://github.com/ros-tooling/cross_compile>`_
+  * `ros-tooling/system_metrics_collector <https://github.com/ros-tooling/system_metrics_collector>`_
+  * RQt
+
+    * `ros-visualization/python_qt_binding <https://github.com/ros-visualization/python_qt_binding>`_
+    * `ros-visualization/qt_gui_core <https://github.com/ros-visualization/qt_gui_core>`_
+    * `ros-visualization/rqt <https://github.com/ros-visualization/rqt>`_
+    * `ros-visualization/rqt_action <https://github.com/ros-visualization/rqt_action>`_
+    * `ros-visualization/rqt_console <https://github.com/ros-visualization/rqt_console>`_
+    * `ros-visualization/rqt_graph <https://github.com/ros-visualization/rqt_graph>`_
```

Comment:
I'd link to to the repository page on index.ros.org instead of github directly. The repos sometimes move, but IIUC if the source entry is updated in `ros/rosdistro` then the repository page at index.ros.org will have a link to the new location.

https://index.ros.org/r/rqt_graph/

Reply:
Done as part of 169e87d.